azure-pipelines.yml

```diff
@@ -54,6 +54,18 @@ pr:
     include:
       - '*'
 
+schedules:
+  - cron: "0 8 23-29 * 0"
```

Comment:
Suggest:
- cron: "0 8 22-28 * 0" # Fourth Sunday of each month at 8:00 UTC

(Relaying feedback I got from my similar PRs): The comment is nice so looking up the cron syntax isn't as necessary; the 22-28 is the 4th Sunday, 23-29 is the... last Sunday except when the last Sunday is the 30th, and doesn't run at all in February if February 22nd is the last Sunday in February (2026).   :-)